posthog/warehouse/models/snapshot_config.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+from posthog.models.team import Team
+from posthog.models.utils import CreatedMetaFields, DeletedMetaFields, UUIDTModel
+
+
+class DataWarehouseSnapshotConfig(CreatedMetaFields, UUIDTModel, DeletedMetaFields):
+    team = models.ForeignKey(Team, on_delete=models.CASCADE)
+    saved_query = models.OneToOneField("posthog.DataWarehouseSavedQuery", on_delete=models.CASCADE)
+    config = models.JSONField(default=dict)
```

Comment:
Do we need this as a single `JSON` config column? Now it has its own table, we can split the config column into properly structured and typed columns instead